docs/index.html

```diff
@@ -0,0 +1,737 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!-- Google Tag Manager -->
+    <!-- https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/6354 -->
+    <script>
+      (function(w, d, s, l, i) {
+        w[l] = w[l] || [];
+        w[l].push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+        var f = d.getElementsByTagName(s)[0],
+          j = d.createElement(s),
+          dl = l != "dataLayer" ? "&l=" + l : "";
+        j.async = true;
+        j.src = "https://www.googletagmanager.com/gtm.js?id=" + i + dl;
+        f.parentNode.insertBefore(j, f);
+      })(window, document, "script", "dataLayer", "GTM-T2ZTDXZ");
+    </script>
+    <!-- End Google Tag Manager -->
+    <title>VSP Documentation Landing Page</title>
+    <meta charset="UTF-8" />
+    <link
+      href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <!-- Google Tag Manager (noscript) -->
+    <noscript
+      ><iframe
+        src="https://www.googletagmanager.com/ns.html?id=GTM-T2ZTDXZ"
+        height="0"
+        width="0"
+        style="display:none;visibility:hidden"
+      ></iframe
+    ></noscript>
+    <!-- End Google Tag Manager (noscript) -->
+
+    <header class="h-32 px-16 pt-8 bg-blue-900 text-white">
+      <div class="pb-3">
+        <span class="text-4xl">VA.gov | Documentation</span>
+      </div>
+    </header>
+
+    <main>
+      <section>
+        <div class="bg-gray-200 px-40 py-16">
+          <h1 class="text-3xl font-bold text-gray-800">
+            A knowledge hub for building on VA.gov
+          </h1>
+          <div class="pt-4">
+            Connecting you to documentation, tools, and resources to help you as
+            you work.
+          </div>
+        </div>
+        <div class="beta-banner bg-blue-300 px-40 py-4">
+          <span class="font-bold text-xl">
+            This is a beta site.
+          </span>
+          We are always looking to make improvements.
+          <span class="font-bold">
+            Send us your feedback.
```

Comment:
what's this going to do?

Reply:
Content & IA is still figuring out what they want that link to do. So it's just a placeholder for now.